doc/content/sqa/malamute_scs.md

```diff
@@ -1 +1,26 @@
-!template load file=sqa/app_scs.md.template app=MalamuteApp category=malamute
\ No newline at end of file
+!template load file=sqa/app_scs.md.template app=MALAMUTE category=malamute
+
+## Clang Format
+
+!style halign=left
+Like MOOSE, MALAMUTE uses `clang-format` with a customized
+[config file](https://github.com/idaholab/malamute/blob/devel/.clang-format)
+for code formatting. If you have clang installed, you can run
+
+```
+git clang-format [<branch>]
+```
+
+to automatically format code changed between your currently checked-out branch
+and `<branch>` (if left out, it defaults to the `HEAD` commit). If you don't do
+this before submitting your code, don't worry! The continuous integration
+testing system, [CIVET](https://civet.inl.gov), that is triggered when
+you submit a pull request will check your code and provide information on the
+changes needed to conform to the code style (if any).
+
+## MOOSE Code Standard
+
+!style halign=left
+For extended information on file guidelines, naming conventions, example code,
+doxygen documentation, and other tips, please see
+[the MOOSE Code Standards](sqa/framework_scs.md).
```

Comment:
Is MOOSE correct here or should it be MALAMUTE? Maybe the header title should be MALAMUTE and then we say something about how it follows MOOSE code standards?